controller/build.go

```diff
@@ -236,6 +236,8 @@ func PostBuild(c *gin.Context) {
 	build.Started = 0
 	build.Finished = 0
 	build.Enqueued = time.Now().UTC().Unix()
+	build.Event = c.DefaultQuery("event", build.Event)
+	build.Deploy = c.DefaultQuery("deploy_to", build.Deploy)	
 	for _, job := range jobs {
```

Comment:
we should probably also only enable overriding `build.Event` and `build.Deploy` if `fork` is true. The reason being when `fork` is false it just re-runs the existing build number and does not create a new build number. We probably want to treat the event end deploy values as immutable unless being forked.